crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -733,6 +802,7 @@ public BulkResponse bulkRequest(BulkRequest requests) {
             Error.pop();
             return response;
         } finally {
```

Comment:
I see. I guess the REST layer takes care of any errors there, so we can track that there.